metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
+    article = models.ForeignKey(Article)
+    topic = models.ForeignKey(Topic)
+    owner = models.ForeignKey(User)
+
+    location = models.IntegerField()
```

Comment:
Yes, that's the character index. It was the only general enough thing we could say. ID of the container could work but then it's really website implementation specific. I'd keep it this way and then update it once we know what kind of info we can get from the original articles.